src/arch/xen/boot.S

```diff
@@ -1,42 +1,40 @@
 /**
  * @file
- * @brief bootstrap the kernel
+ * @brief bootstrap the Xen kernel
  *
  * @date 10.11.10
  * @author Nikolay Korotky
+ * @author Andrey Golikov
+ * @author Anton Kozlov
  */
 
-
-#include <asm/linkage.h>
 #include <hal/cpu.h>
 
-	.section .text
-	.align 4
+#define C_ENTRY(name) .globl name; .align 2; name
```

Comment:
This derived from x86 without modification. Seems like 2-align originates from old x86 align requirements. OK, I'll fix it